documentation/website/docs/installation.md

```diff
@@ -71,6 +72,20 @@ $ source ~/.bashrc
 ```
 You should be able to open a new shell and run `pyre -h` now, confirming `pyre` was set-up correctly. Any changes made to the Pyre Python client code should be immediately observable the next time you invoke `pyre`
 
+#### Testing changes for Plugin Development
+VSCode will not pick up your shell aliases, so the alias step in the previous section will not work if you're doing VSCode Plugin development. To get around this, instead of creating an alias, we can create an executable script called `pyre` and place it in a directory in our `PATH`:
+
+```bash
+#!/bin/bash
+export PYTHONPATH="/path/to/pyre-check:$PYTHONPATH" python -m client.pyre "$@"
+```
+Add the `pyre-check/scripts` directory to `PATH` (assuming you placed the above script in that directory) and then use the command `pyre` to launch the client like before
+
+```bash
+$ echo 'export PATH="/path/to/pyre-check/scripts:$PATH"' >> ~/.bashrc
```

Comment:
nitpick here: we don't need the `export` here either. `PATH` is always exported, otherwise, well, nothing would work.